.github/workflows/process-notifications.yml

```diff
@@ -1,41 +1,50 @@
 name: Process Notifications
-
 on:
   workflow_call:
     inputs:
-        environment:
-          description: The Github environment to load secrets from
-          type: string
-          required: true
+      environment:
+        description: The Github environment to load secrets from
+        type: string
+        required: true
+      force-deploy:
```

Comment:
why do we need force deploy on this script? is this trying to inherit from the deploy workflow?

These env vars are feeling brittle; is there a way we can control this logic from the deploy workflow more directly?

Reply:
It is to skip the notifications workflow if the current deploy is a forced one.